histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -384,7 +384,12 @@ def main(args):
 
     annotation = {
         'name': annot_fname + '-nuclei-' + args.nuclei_annotation_format,
-        'elements': nuclei_list
+        'elements': nuclei_list,
+        'attributes': {
+            'src_mu_lab': src_mu_lab,
```

Comment:
It looks like `src_mu_lab` and `src_sigma_lab` are either `None` or a numpy array.  As a numpy array, they don't json serialize, so we probably have to do something like `'src_mu_lab': None if src_mu_lab is None else src_mu_lab.to_list(),`

Reply:
Yeah, they are a numpy array with 3 elements. I update these changes.